.golangci.yml

```diff
@@ -9,19 +9,82 @@ run:
   go: '1.19'
 
 linters:
-  disable-all: true
-  enable:
-    - gofmt
-    - goimports
-    - unused
-    - errcheck
-    - gosimple
-    - gocognit
-    - dupword
-    - nilerr
-    - whitespace
-    - misspell
-    - errname
+  enable-all: true
+  disable:
+    # Deprecated linters:
+    - deadcode
+    - exhaustivestruct
+    - golint
+    - govet
+    - ifshort
+    - interfacer
+    - maligned
+    - nosnakecase
+    - scopelint
+    - structcheck
+    - varcheck
+
+    # Disabled for now:
+    - asasalint
+    - bodyclose
+    - containedctx
+    - contextcheck
+    - cyclop
+    - depguard
+    - dogsled
+    - dupl
+    - durationcheck
+    - errorlint
+    - exhaustive
+    - exhaustruct
+    - forbidigo
+    - forcetypeassert
+    - funlen
+    - gci
+    - gochecknoglobals
+    - gochecknoinits
+    - goconst
+    - gocritic
+    - gocyclo
+    - godot
+    - godox
+    - goerr113
+    - gofumpt
+    - gomnd
+    - gomoddirectives
+    - inamedparam
+    - interfacebloat
+    - ireturn
+    - lll
+    - maintidx
+    - makezero
+    - mirror
```

Comment:
This will conflict with https://github.com/prysmaticlabs/prysm/pull/13342

Reply:
Good pointing out. Removed that from the list when resolving conflicts.

Reply:
Also removed usestandlibvars. Until #13342 is merged, this will fail lint checks.